code/game/objects/items/weapons/boomerang.dm

```diff
@@ -32,6 +32,7 @@
 
 	starting_materials = list(MAT_PLASTIC = 1200)
 	melt_temperature = MELTPOINT_PLASTIC
+	w_type = RECYK_PLASTIC
```

Comment:
why not make the parent wood type, since it's made of wood?

Reply:
Done. Six sheets, per the stack recipe